lib/js/src/session/SdlSession.js

```diff
@@ -203,8 +215,8 @@ class SdlSession {
      * @param {Number} sessionId - represents a byte 
      * @param {Boolean} isEncrypted
      */
-    startService (serviceType, sessionId, isEncrypted) {
-        this._sdlProtocol.startService(serviceType, sessionId, isEncrypted);
+    startService (serviceType, sessionId, isEncrypted, bsonPayload) {
```

Comment:
Not sure if this is necessary? Seems that `SdlSession.startService` does not include a param for `bsonPayload` in the Java Suite.